catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/AccessControlAccessPluginTest.java

```diff
@@ -137,6 +138,62 @@ public void testBasicUpdateFailsWithMissingAdmin() throws Exception {
     accessPlugin.processPreUpdate(update, ImmutableMap.of(MOCK_METACARD_ID, before));
   }
 
+  @Test
+  public void testSuccessWhenWriteableUserModifies() throws StopProcessingException {
+    Metacard before =
+        AccessControlUtil.metacardFromAttributes(
+            ImmutableMap.of(
+                Core.ID,
+                MOCK_METACARD_ID,
+                Core.METACARD_OWNER,
+                "before",
+                SecurityAttributes.ACCESS_INDIVIDUALS,
+                ImmutableList.of(MOCK_IDENTITY_ATTR)));
+
+    Metacard after =
+        AccessControlUtil.metacardFromAttributes(
+            ImmutableMap.of(
+                Core.ID,
+                MOCK_METACARD_ID,
+                Core.METACARD_OWNER,
+                "before",
+                Core.LOCATION,
+                "Arizona",
+                SecurityAttributes.ACCESS_INDIVIDUALS,
+                ImmutableList.of(MOCK_IDENTITY_ATTR)));
+
+    UpdateRequest update = mockUpdateRequest(ImmutableMap.of(MOCK_METACARD_ID, after));
+    accessPlugin.processPreUpdate(update, ImmutableMap.of(MOCK_METACARD_ID, before));
+  }
+
+  @Test(expected = StopProcessingException.class)
+  public void testFailureWhenNonWriteableUserModifies() throws StopProcessingException {
+    Metacard before =
```

Comment:
✏️ Could refactor the metacards `before` and `after` as globals or create a method that creates these metacards so that you don't have to explicitly create a new one for every test.

Reply:
I want the metacards to be different for very specific edge cases, so I didn't want to genericize any metacard creation.